patches/minecraft/net/minecraft/network/play/server/SPacketChunkData.java.patch

```diff
@@ -0,0 +1,18 @@
+--- ../src-base/minecraft/net/minecraft/network/play/server/SPacketChunkData.java
++++ ../src-work/minecraft/net/minecraft/network/play/server/SPacketChunkData.java
+@@ -140,6 +140,7 @@
+         if (this.func_149274_i())
+         {
+             p_189555_1_.writeBytes(p_189555_2_.func_76605_m());
++            int localBiomeCount = p_189555_2_.getLocalBiomes().size(); p_189555_1_.writeByte((byte)(localBiomeCount & 255)); Short[] localBiomeIds = p_189555_2_.getLocalBiomeIds(); for (int k = 0; k < localBiomeCount; k++) p_189555_1_.writeShort(localBiomeIds[i]);
```

Comment:
lolwot, you don't need to squish it to one line, make it readable.

Reply:
Actually the official guide for contributing says the opposite: https://github.com/MinecraftForge/MinecraftForge/wiki/If-you-want-to-contribute-to-Forge 

"In spirit of the previous point: prevent to add more lines than necessary. If you can reduce your statement to one single line, please do so. "

They actually use an example where they basically say that it will reduce readability.  And even though they say "statement" their example is a nested if statement that they squash.

Reply:
I never really understood that guideline. For merging it shouldn't matter how many lines you add---in one chunk. Merging a 1000-line addition into the right location is just the same as merging a 1-line addition to that location and about 1/1000th the work of merging 1000 1-line additions.